lib/spack/spack/test/cc.py

```diff
@@ -278,8 +278,8 @@ def test_ld_flags(wrapper_environment, wrapper_flags):
         ld,
         test_args,
         ["ld"]
-        + spack_ldflags
         + test_include_paths
+        + ["".join(spack_ldflags)]
```

Comment:
Why the join here? Is this not a list of flags already?

Reply:
The concern is that if there's more than one, like the list is `['-l','meh','-L','/bah']` then wouldn't this break pretty badly?

Reply:
In the wrapper itself, it will be correct and there will be no issues.
It may affect only this test if this test itself will be updated one day, but probably it can be resolved when(if) it happens